libkbfs/md_journal.go

```diff
@@ -198,32 +234,77 @@ func (j mdJournal) mdDataPath(id MdID) string {
 	return filepath.Join(j.mdPath(id), "data")
 }
 
+func (j mdJournal) mdInfoPath(id MdID) string {
+	return filepath.Join(j.mdPath(id), "info.json")
+}
+
+// mdInfo is the structure stored in mdInfoPath(id).
+type mdInfo struct {
```

Comment:
Hmm, can't, since we're using JSON. Added a TODO, though...should be possible to add a helper function or something.